src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/hooks/sortFunctions.js

```diff
@@ -113,10 +119,11 @@ const sortStatus = (clientValueA: Object, clientValueB: Object, direction: strin
     return 0;
 };
 
-const sortDataFromEvent = ({ dataA, dataB, type, columnName, direction }: Object) => {
+export const sortDataFromEvent = ({ dataA, dataB, type, columnName, direction }: Object) => {
     if (!type) {
         log.error(errorCreator('Type is not defined')({ dataA, dataB }));
     }
+    console.log('direction', direction);
```

Comment:
Can you please remove this forgotten `console`? Other than that, the bug fix seems good to me.